src/pathTemplate.ts

```diff
@@ -122,6 +122,8 @@ export class PathTemplate {
    *   parsed
    */
   render(bindings: Bindings): string {
+    console.warn('bindings: ', bindings);
```

Comment:
no debug logs please :)

Reply:
sure, I'm using this branch to test monitoring now. Will remove this once the samples-test in monitoring passes! Thanks!